### DISCUSSION TITLE
Extension to persist form fields to different storages

### DISCUSSION THREAD

**Discussion by jyrimatti** [bigskysoftware/htmx#2403] [Show and tell]
Hi,
I made an extension to persist state with different needs. It feels useful, since it allowed me to remove quite a lot of state handling code from a small application of mine.

What do you think?
Any suggestions for improvement?

https://github.com/jyrimatti/htmx-persist-fields

**Comment by nickchomey** [comment]
It seems to me that you could incorporate a service worker and background sync and it would allow for offline form submission with automatic sync when back online
